mne/channels/tests/test_montage.py

```diff
@@ -122,7 +123,24 @@ def test_montage():
     fname = op.join(tempdir, kind)
     with open(fname, 'w') as fid:
         fid.write(input_str)
-    montage = read_montage(op.join(tempdir, 'test_fid.hpts'), transform=True)
+    montage = read_montage(op.join(tempdir, kind), transform=True)
```

Comment:
could you add the test for the `.hpts`? the changes below fixes the test for `.sfp` but doesn't check the former.